csv/SMT Dx2 Database - Skills.csv

```diff
@@ -34,7 +34,7 @@ Element,Name,JP Name,Cost,Description,Target,Skill Points,Learned By,Transferabl
 "phys","Ouas","ウアス","6 MP","Inflicts Phys (Physical) damage with 50% crit rate and 160 power on a Single Enemy, while healing caster 40% of damage dealt.\nLv2: +10% to Phys hit rate.\nLv3: +20% to Phys hit rate.\nLv4: +20% to Phys hit rate. +15% to damage.","Single Enemy","","",""
 "phys","Gungnir","グングニル","7 MP","Inflicts Phys (Physical) damage with 30% crit rate and 145 power on All Enemies.\nLv2: +10% to Phys attack hit rate.\nLv3: +15% to Phys attack hit rate.\nLv4: +20% to Phys attack hit rate. +15% to damage.","All Enemies","","",""
 "phys","Mist Slashes","乱・かすみ斬り","6 MP","Inflicts Phys (Physical) damage 3 to 5 times with 50 power on Random Enemy/(ies).","Random Enemies","","",""
-"phys","Gae Bolg","ゲイボルグ","7 MP","Inflicts Phys (Physical) damage 4 to 6 times with 50 power on Random Enemy/(ies).","Random Enemies","","",""
+"phys","Gae Bulg","ゲイボルグ","7 MP","Inflicts Phys (Physical) damage 4 to 6 times with 50 power on Random Enemy/(ies).\nLv2: +10% to Phys attack hit rate.\nLv3: +20% to Phys attack hit rate.\nLv4: +40% to Phys attack hit rate.","Random Enemies","","",""
```

Comment:
Make sure you update the demon.csv with this information when a skill name changes so the demons skills are correct.